src/server/pfs/cmds/cmds.go

```diff
@@ -74,7 +75,16 @@ or type (e.g. csv, binary, images, etc).`,
 	}
 	commands = append(commands, cmdutil.CreateDocsAliases(repoDocs, "repo", " repo$", repos))
 
-	var description, project string
+	cfg, err := config.Read(false, true)
```

Comment:
Could we do this at a higher level, maybe even the main function? It seems like it would be better to do this in one place and pass around a config and / or context to be used by the commands. That would also let us avoid having to handle errors when creating the commands.

Reply:
Refactoring for _just_ PFS & PPS commands turned out to be extremely simple. I started to do the same for the rest of the codebase (e.g. for transactions), but it gets pretty hairy because some parts of the code expect a read-only config, others a writable one, others expect an error when getting the active context and others don’t. Resolving all that is a bigger story. How do you feel about me opening an issue on the backlog for that?

Reply:
Yeah, that makes sense.

Reply:
Thanks, I created CORE-1281 for that.